### ISSUE TITLE
[Docs]: Unknown @http.throws(ClassType, string) decorator

### ISSUE THREAD

**Issue by hanayashiki** [deepkit/deepkit-framework#249]
The documentation is here

https://deepkit.io/documentation/framework/http/controller#response

but it is no longer in the code:

https://github.com/deepkit/deepkit-framework/blob/d1533ff6e5a008cb9080a7607e5b7bcda2cefdde/packages/http/src/decorator.ts#L187

**Comment by TheNightmareX** [comment]
Yeah there are many mistakes in the current document. There is a new document coming soon, you can view its preview version at https://deepkit-book.herokuapp.com/

**Comment by marcj** [comment]
Yes, although I think http.throw can be removed entirely
